openstates/cli/update.py

```diff
@@ -351,7 +355,11 @@ def _tb_info(type, value, tb):  # type: ignore
 
         sys.excepthook = _tb_info
 
+    logging.info(f"Module: {args.module}")
     juris, module = get_jurisdiction(args.module)
+    logging.info(f"Juris: {juris}")
+    logging.info(f"Module: {module}")
```

Comment:
Could you remove these logging lines if they are no longer needed?

Reply:
Done